### DISCUSSION TITLE
feed-in price does not track retail price

### DISCUSSION THREAD

**Discussion by Rujo** [amberelectric/public-api#198] [General]
I have been an Amber customer for the last 2 weeks. 
I am using the API to track both the feed-in tariff and the retail price.
The price jump of the period from 2pm to 9 pm is what makes Amber so interesting for battery owners.

All went well until yesterday.

But today I was shocked to see that the feed-in tariff does not track the retail price anymore (with about a 15 cent/kWh difference), but the feed-in tariff stays essentially at the level of the wholesale price.
Surprisingly, this happens only on my account and not on a friend's Amber account, which I manage.

As I am new to Amber, is there anything I need to do on my side to get the feed-in tariff tracking the retail price?

Below are two relevant pieces of data from the API.

![image](https://github.com/amberelectric/public-api/assets/8346461/8f060a5f-62ef-4235-9e40-d58cb50452fd)
![image](https://github.com/amberelectric/public-api/assets/8346461/945a52f4-8c1f-41eb-9013-fe1c26eb6af6)

**Comment by Rujo** [comment]
correction of typo:
The price jump of the period from 2pm to 8pm is what makes Amber so interesting for battery owners.

**Comment by Rujo** [comment]
This is the chart that I generated from the API
![image](https://github.com/amberelectric/public-api/assets/8346461/09a0208b-91de-436c-a6d0-d265623d5c62)

**Comment by madpilot** [comment]
The feed in price is always basically the wholesale price unless you are on a two-way tariff or a premium feed in tariff (the slight difference is due to loss factors).

Are you expecting to be on a two-way tariff?

**Reply by Rujo** [reply]
Yes, I thought that was the normal situation.
How can I get the two-way tariff?

**Reply by madpilot** [reply]
Email info@amber.com.au and request a change of tariff
